lib/earmark/earmark_parser_proxy.ex

```diff
@@ -1,25 +1,25 @@
-defmodule Earmark.EarmarkParserProxy do
+defmodule Earmark.Earmark.ParserProxy do
```

Comment:
I couldn't help but notice, shouldn't this be `Earmark.ParserProxy` (without the double `Earmark`)? @josevalim
(Sorry if I'm off, I'm not familiar with the project)

Reply:
Yeah, good find. I can submit a PR. :) Although perhaps this module can be removed too. :)

Reply:
Or perhaps, since it is publicly documented, we should not change its name.

Reply:
I just merged it I did not understand why. I will be little available in the next days, shall I ask Dave to give you or some proxy of you (pun intended) commit rights here and also publish rights to hex.

I will merge and publish as much as I can, but maybe it would be good to have fallback options.

Again thank you for helping out.

Reply:
Thank you for that explanation.